_posts/2024-08-24-GSoC2024-dingo.md

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
Thanks @SotirisTouliopoulos! That looks good. A general comment is that the text is more like a gsoc final submission. I would make a few modifications so that it would be more like a blog-post.